src/main/resources/application.yaml

```diff
@@ -121,32 +121,38 @@ containers:
     - container: sscs
       jurisdiction: SSCS
       poBox: 12626
+      enabled: true
```

Comment:
Is it possible to have true as the default value? As far as I remember we discussed this

Reply:
Updated code to set the values as true by default